R/semanticPage.R

```diff
@@ -91,6 +95,13 @@ check_semantic_theme <- function(theme_css) {
 #'
 #' This creates a Semantic page for use in a Shiny app.
 #'
+#' It makes use of two important options:
+#' \code{shiny.minified} with a logical value, tells whether it should attach min or full
+#' semnatic css or js (TRUE by default).
+#' \code{shiny.custom.semantic} if this option has not NULL character semanticPage
+#' will take dependencies from custom css file specified in this path
```

Comment:
Please add info about js: `custom css` ->`custom css and js files`.

Maybe it's worth to notice that shiny.custom.semantic directory should have both normal and minified version of above files (this is up to you).